features/step_definitions/fixtures_steps.rb

```diff
@@ -44,9 +44,8 @@
 end
 
 Given(/^"([^"]*)" has been added as an MenuItem to "([^"]*)"$/) do |child, parent|
-  #binding.pry
   menu = Menu.find_by(title: parent)
   menu_item = MenuItem.find_by(name: child)
-  menu.menu_items << menu_item
+  MenuItemsMenu.create(daily_stock: 10, menu_item: menu_item, menu: menu)
```

Comment:
What is the difference with the previous way of adding the `menu_item` to this `menu`?